seeder-start-restart.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Start-restart script for Kiwix-seeder
+#
+# Assumes an optional (yet recommended) config file in /etc/seeder.config
+# to overwrite the following variables.
+
+CONTAINER_NAME="seeder"                             # name of docker container
+IMAGE="ghcr.io/kiwix/bittorrent-seeder:latest"      # docker image to use
+
+DATA_PATH=$(pwd)/kiwix-seeder                       # path to store ZIM files (and incomplete .!qB ones) in (there's no hierarchy)
+MAX_STORAGE="10GiB"									# maximum disk-space to use
+SLEEP_INTERVAL="5m"                                 # how long to pause in-between catalog checks when using --loop
```

Comment:
Shouldn't we have a bigger sleep interval by default ? I feel like 1h is already a lot, maybe even 1 day should be advised as a default value to avoid flooding our servers for a very limited added value (geeks always want the most up-to-date information, so let's try to avoid pushing in their direction ^^).